src/schemas/json/ocelot.json

```diff
@@ -42,12 +42,12 @@
           "AddHeadersToRequest": {
             "$id": "#/properties/Routes/items/properties/AddHeadersToRequest",
             "type": "object",
-            "title": "The Addheaderstorequest Schema"
+            "title": "The AddHeadersToRequest Schema"
```

Comment:
My naive understanding/hope is that this is a reasonable change, I believe it's just a user facing title